README.adoc

```diff
@@ -373,9 +381,9 @@ them on the Zipkin server.
 
 == Great work! You're done!
 
-You have just utilized MicroProfile OpenTracing to customize how and what tracers are delivered to Zipkin.
+You have just utilized MicroProfile OpenTracing to customize how and which tracers are delivered to Zipkin.
 
-Feel free to try one of the related MicroProfile guides. They demonstrate new technologies that you can learn and
-expand on top what you built here.
+Feel free to try one of the related MicroProfile guides. They demonstrate new technologies that you
+can learn to expand on top of what you built here.
```

Comment:
What about saying "additional" instead of "new."? The word "new" can be such a subjective term.

Reply:
Yeah you're right, it does sound a bit weird. I changed it to "additional"